lib/components/package.json

```diff
@@ -37,7 +37,8 @@
     "*.d.ts"
   ],
   "scripts": {
-    "prepare": "ts-node ../../scripts/prebundle.ts"
+    "prepare": "ts-node ../../scripts/prebundle.ts",
+    "check": "tsc --noEmit"
```

Comment:
not sure if you guys have something similar, but this is a command I add to every package to make sure types are correct.

Reply:
yeah, I want to add that to all packages indeed!